src/ui/public/agg_types/metrics/bucket_avg.js

```diff
@@ -0,0 +1,18 @@
+import AggTypesMetricsMetricAggTypeProvider from 'ui/agg_types/metrics/metric_agg_type';
+import { makeNestedLabel } from './lib/make_nested_label';
+import SiblingPipelineAggHelperProvider from './lib/sibling_pipeline_agg_helper';
+
+export default function AggTypeMetricDerivativeProvider(Private) {
+  const MetricAggType = Private(AggTypesMetricsMetricAggTypeProvider);
+  const siblingPipelineHelper = Private(SiblingPipelineAggHelperProvider);
+
+  return new MetricAggType({
+    name: 'avg_bucket',
+    title: 'Average Bucket',
+    makeLabel: agg => makeNestedLabel(agg, 'overall average'),
+    group: 'Pipeline Aggregations',
```

Comment:
The string "Pipeline Aggregations" is repeated in many places.  What about creating a metrics_constants file to put it in?  Then if we want to change the string we only have to do it in one spot.